frameworks/engineering/backend.md

```diff
@@ -206,6 +206,11 @@ topics:
           - criteria: "Implements complex RPC handlers"
             examples:
               - "Implements complex business logic, orchestrates multi-stage processes, deals with idempotency or distributed data sources"
+          - criteria: "Proactively considers security implications of their work"    
```

Comment:
A good question to ask when writing these behaviours is to try to articulate what Amazing vs Good vs Bad looks like this for behaviour and what would evidence that. It's hard to explain to someone how they need to show that they _consider_ things.

Any thoughts?

Reply:
Sam and I went back and forth about "considers" here (we paired on the original language).  Ultimately we compared it against the other time it's used in the framework:  "considers metrics when designing new systems." 

I think "considers" is more appropriate here than there, and with the examples given below that illustrate the various ways someone could demonstrate that they've given thought to (or "considered") security, I think it's acceptable.

All ears if we can come up with better language, though.  I think "considers" is overall a weak verb for the framework, for the reasons Will highlighted.

Reply:
Thanks for your thoughts! Totally agree that _considers_ can be a little hard to evidence at first glance. I think what we're trying to get at with the word _considers_  is that people have security in their heads when writing code, proposals, and when dealing with data . Some examples of evidence I can think of:

- Proposal which evaluate the security risks of the implementation, or leaving comments on another proposal
- Code comments, PR comments that raise security concerns - e.g. adding rate limiting in an API, or leaving a comment to explain why we choose not to
- Blacklisting RPCs they encounter which should ideally not be callable by all engineers
- Questioning and challenging processes that involve data handling - e.g. automating feature flag enabling when testing a new feature
- Coming across something that "doesn't seem right" and dropping a message to security in #security-requests (or by DMing the floater if it's sensitive)

@JacePro @wpillar what are your thoughts?

Reply:
>Proposals which evaluate the security risks of the implementation, or leaving comments on another proposal
>Code comments, PR comments that raise security concerns - e.g. adding rate limiting in an API, or leaving a comment to explain why we choose not to

Add these two as examples and I think that will be sufficient.